t/sys/creat64.c

```diff
@@ -47,15 +46,16 @@ int creat64_test(char* unifyfs_root)
     ok(fd >= 0, "creat64 non-existing file %s (fd=%d): %s",
        path, fd, strerror(errno));
 
-    rc = close(fd);
+    ok(close(fd) != 1, "close() worked");
```

Comment:
@tonyhutter, I have been doing some refactoring of the testing suite (in a PR coming later this week) to remove repetition and adding convenience functions as well as new tests and ended up doing the opposite with a lot of things like this. That is, removing `rc` and just turning it into `close(fd);` rather than putting it in a test.

Since, in this case, `close()` is already tested in [creat-close.c](https://github.com/LLNL/UnifyFS/blob/dev/t/sys/creat-close.c) and will show up there first if it breaks, is there an advantage I'm not thinking of to testing it again in latter testing files like this?
For context, my thoughts were that, in turning this into another test, we end up with a large number of our overall tests being repeated `open()` or `close()` tests which can clutter the results and make it look like we have more tests that we we actually do. Also can help keep the tests logically separated.

That said, it looks like we're getting some compiler warnings that are making travis fail for doing a `write()` but not asking for the result back. This is for tests where calls like `open()`, `write()`, and `close()` are simply used to set up the testing environment to test the function/wrapper in question.

I'm not sure the best approach here and would like any thoughts you have (I'm getting the feeling I've spent too much time thinking about the testing suite 😆).

Reply:
As stated on the call, let's do what you've done here with our other tests and I'll do this for future ones as well. Since I'm touching a lot of the testing files this week, I can always add `-Werror` to the testing Makefile with my PR. In there I'll fix the warnings for writing but not asking for a result back.